tests/test_default_gaussian.py

```diff
@@ -394,14 +394,14 @@ def test_apply_errors(self):
             p = [thermal_state(0.5)]
             self.dev.apply('GaussianState', wires=[0], par=[p])
 
-        with self.assertRaisesRegex(ValueError, 'incorrect number of subsystems'):
+        with self.assertRaisesRegex(ValueError, 'Incorrect number of subsystems'):
             p = U
             self.dev.apply('Interferometer', wires=[0], par=[p])
 
-        with self.assertRaisesRegex(ValueError, 'Only 2-mode interferometers are currently supported.'):
+        with self.assertRaisesRegex(ValueError, 'Bad target subsystems'):

```

Comment:
If you take suggestion for new error message above, will need to update this assert

Reply:
```suggestion
        with self.assertRaisesRegex(ValueError, "Invalid target subsystems provided in 'wires' argument"):
```